pandas/io/tests/test_parsers.py

```diff
@@ -2984,6 +2984,28 @@ def test_variable_width_unicode(self):
         tm.assert_frame_equal(expected, read_fwf(BytesIO(test.encode('utf8')),
                                                  header=None, encoding='utf8'))
 
+    def test_convert_to_nd_arrays(self):
+        #GH 9266
+        with open('test.txt','w') as f:
+            f.write(
+            """1421302964.213420    PRI=3 PGN=0xef00      DST=0x17 SRC=0x28    04 154 00 00 00 00 00 127
+               1421302964.226776    PRI=6 PGN=0xf002               SRC=0x47    243 00 00 255 247 00 00 71"""
```

Comment:
I think the issue is that this line starts with extra whitespace, throwing off `colspecs` below.